### PR TITLE
:recycle: refactor: merge websocket middleware to contrib repo

### PR DESCRIPTION
**Pull request by efectn**
Same reason as https://github.com/gofiber/fiber/commit/c7c37d9b504f530a23ee1b03df872e8e0f17002f

### CODE REVIEW

**Comment by ReneWerner87** [comment]
thx

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -58,3 +58,5 @@ jobs:
                 run: cd ./fiberi18n && go test ./... -v -race
             -   name: Test fiberzerolog Middleware
                 run: cd ./fiberzerolog && go test ./... -v -race
+            -   name: Test websocket Middleware
```
Comment (gaby) [inline .github/workflows/test.yml:61]: Can we add go1.20 to the test matrix here too?

File: websocket/go.mod [hunk 2]
```diff
@@ -0,0 +1,26 @@
+module github.com/gofiber/contrib/websocket
+
+go 1.20
```
Comment (gaby) [inline websocket/go.mod:3]: Should this be go1.20?

**Review by gaby — CHANGES_REQUESTED** [review]

File: websocket/go.mod (hunk 2 above)
Comment (efectn) [reply websocket/go.mod:3]: it's because of i ran go init on go1.20. Has no affect aganist old go versions unless we run go mod tidy with compat

Comment (gaby) [reply websocket/go.mod:3]: Ahhhh, ok

**Review by gaby — APPROVED** [review]

**Pull request merged**